publish/publish.proj

```diff
@@ -164,14 +169,26 @@
                            BlobNamePrefix="Runtime/$(SharedFrameworkNugetVersion)"
                            DownloadDirectory="$(DownloadDirectory)" />
     <ItemGroup>
-      <_DownloadedPackages Include="@(_CoreHostPackages->'$(DownloadDirectory)%(Filename)%(Extension)')" />
+      <_DownloadedPackages Include="@(_CoreHostPackages->'$(PublishDirectory)%(Filename)%(Extension)')" />
       <_DownloadedSymbolsPackages Include="%(_DownloadedPackages.Identity)" 
                                  Condition="$([System.String]::new('%(_DownloadedPackages.Identity)').EndsWith('.symbols.nupkg'))" />
       <_DownloadedStandardPackages Include="@(_DownloadedPackages)"
                                   Exclude="@(_DownloadedSymbolsPackages)" />
     </ItemGroup>
+  </Target>
+
+  <Target Name="SignSymbolPackages" DependsOnTargets="InjectSignedSymbolCatalogIntoSymbolPackages">
```

Comment:
If the Windows Dev Kit or Driver Kit are installed (so makecat is present), everything should be fine - we stub out the SignFiles target that MicroBuild depends on.  We could do something wonky like check for `%ProgramFiles%\Microsoft SDKs\Windows` if blowing up without the WDK is unacceptable - this seems like a weird scenario though.  I'm not sure we should ever be publishing directly from a dev build because it would be putting unsigned bits on the official feeds.